src/validators.ts

```diff
@@ -34,3 +34,10 @@ validators.array = (value: any[], options: ArrayValidatorOptions, key: string, a
 validators.object = (value: any, options: MapValidatorOptions, key: string, attributes: any) => {
     return validate(value, options.constraints);
 };
+
+validators.urlAllowEmpty = (values: any, ...others: any[]) => {
+    if (!values) return;
+    return validators.url(values, ...others);
+};
+
+export default validate;
```

Comment:
Forgot about this one. Wouldn't have worked without this aye?

Reply:
Actually, XXXexists in the repositories also add custom validators. In that case, they should also be exported there and imported here.